### PR TITLE
new: bump 30% gas price.

### PR DESCRIPTION
**Pull request by sanchaymittal**

### CODE REVIEW

File: modules/contracts/src.ts/services/ethReader.ts [hunk 1]
```diff
@@ -497,6 +497,7 @@ export class EthereumChainReader implements IVectorChainReader {
       if (!gasPrice) {
         try {
           gasPrice = await provider.getGasPrice();
+          gasPrice = gasPrice.add(gasPrice.mul(30).div(100))
```
Comment (rhlsthrm) [inline modules/contracts/src.ts/services/ethReader.ts:500]: Move this to the top to be a declared constant.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: modules/contracts/src.ts/services/ethReader.ts (hunk 1 above)
Comment (rhlsthrm) [reply modules/contracts/src.ts/services/ethReader.ts:500]: By "this" I mean the `30`.

Comment (jakekidd) [reply modules/contracts/src.ts/services/ethReader.ts:500]: Should set constant to `0.3`.

File: modules/contracts/src.ts/services/ethReader.ts [hunk 2]
```diff
@@ -40,6 +40,7 @@ import { ChannelFactory, ChannelMastercopy, TransferDefinition, TransferRegistry
 import { Evt } from "evt";
 
 export const MinGasPrice = parseUnits("5", "gwei");
+export const BumpGasPrice = 30; // 30% bump gas price
```
Comment (rhlsthrm) [inline modules/contracts/src.ts/services/ethReader.ts:43]: Let's use the normal convention we use for these constants which would be `MIN_GAS_PRICE` and `BUMP_GAS_PRICE`.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: modules/contracts/src.ts/services/ethReader.ts (hunk 2 above)
Comment (sanchaymittal) [reply modules/contracts/src.ts/services/ethReader.ts:43]: gotchha

**Review by rhlsthrm — APPROVED** [review]

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**